frontend/src/store/useAuthStore.js

```
@@ -55,30 +68,98 @@ export const useAuthStore = create((set) => ({
+    } catch (error) {
+      const errorMsg = error.response?.data?.message || error.message;
+      set({ authError: errorMsg });
+      console.error("Error resetting password:", errorMsg);
+      return { success: false, message: errorMsg };
+    } finally {
+      set({ isResettingPassword: false });
+    }
+  },
+
   logout: async () => {
     const url = import.meta.env.VITE_API_URL + "/auth/logout";
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)